.github/workflows/packages.yml

```diff
@@ -230,6 +230,64 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
         if: github.ref == 'refs/heads/master' && always() && matrix.publish == 'true' && (steps.build.outcome == 'success' || steps.build.outcome == 'skipped') && steps.check.outcome == 'success' && steps.install.outcome == 'success'
+
+      - name: Check dockerfiles directory existence
+        id: check_directory
+        uses: andstor/file-existence-action@v2
+        with:
+          files: "dockerfiles"  
+      - name: check if dockerfies directory not empty 
+        id: NEdockerdir
+        run: |
+          if [ -z "$(ls -A  packages/${{ matrix.package }}/dockerfiles)" ]; then
+          echo "DIR_EMPTY=true" >> $GITHUB_ENV
+          else
+          echo "DIR_EMPTY=false" >> $GITHUB_ENV
```

Comment:
Could you use set outputs instead of environment variables?

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter